.github/workflows/presubmit-compositions.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Google LLC
```

Comment:
this is a super nit and it doesn't matter because  we don't have a linter that catches these 😝 
```suggestion
# Copyright 2024 Google LLC
```